docs/pre-release/doc-links.yml

```diff
@@ -32,6 +34,8 @@
        link: howtos/outbound
      - title: Send requests to an intercepted service
        link: howtos/request
+     - title: Use Telepresence with a cluster hosted in a VM on the workstation
```

Comment:
How about "Use Telepresence with a local Kubernetes cluster" (I know that is less technically accurate but I don't think most of our users understand how k3s works in a VM for example)